mcp_zammad/client.py

```
@@ -152,6 +153,52 @@ def _parse_bool_env(env_var: str) -> bool:
+    def _graphql_auth(self) -> tuple[dict[str, str], tuple[str, str] | None]:
+        """Build authentication headers or a basic-auth tuple for GraphQL requests."""
+        headers: dict[str, str] = {"Content-Type": "application/json"}
+        if self.http_token:
+            headers["Authorization"] = f"Token token={self.http_token}"
+            return headers, None
+        if self.oauth2_token:
+            headers["Authorization"] = f"Bearer {self.oauth2_token}"
+            return headers, None
+        if self.username and self.password:
+            return headers, (self.username, self.password)
+        raise ConfigException("No credentials available for GraphQL request")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)